web-console/src/lib/compositions/analytics/useUpdateProgram.ts

```diff
@@ -0,0 +1,78 @@
+import { EntitySyncIndicatorStatus } from '$lib/components/common/EntitySyncIndicator'
```

Comment:
this seems unmodified, just moved.
not easy to check.

Reply:
It is unmodified. Was better to move it out, one PR or another